Src/ILGPU.Tests/ArrayViews.cs

```diff
@@ -85,11 +85,39 @@ public void ArrayViewLeaIndex(int length)
             Verify(buffer, expected);
         }
 
+        internal static void ArrayViewLongLeaIndexKernel(
```

Comment:
QUESTION: Spelling?

```suggestion
        internal static void ArrayViewLongLengthIndexKernel(
```

Reply:
I guess this should be okay since this test essential checks the `LoadElementAddress` (LEA) instruction using 64-bit (long) indexers.

Reply:
Ah, Lea = LoadElementAddress. Nevermind, name is fine :+1: